tests/posix/mt/mt.cpp

```diff
@@ -44,13 +44,12 @@ static PMEMfilepool *global_pfp;
 
 class mt : public pmemfile_test {
 public:
-	long ncpus;
-	pthread_t *threads;
+	unsigned ncpus;
+	std::list<std::thread> threads;
```

Comment:
Yeah, to make the code more clear, it could be `emplace_back` instead of `push_back`